translations/de.yaml

```diff
@@ -21,7 +21,8 @@ alexandria:
     no-permission: "Sie haben keine Berechtigung, diese Aktion auszuführen."
     move-document: "Beim Verschieben {count, plural, one {des Dokumentes} other {von # Dokumenten}} ist ein Fehler aufgetreten"
     convert-pdf: "Während dem Umwandeln ist ein Fehler aufgetreten."
-    invalid-file-type: "In der Kategorie \"{category}\" können nur {types} hochgeladen werden."
+    invalid-file-type: 'In der Kategorie "{category}" können nur {types} hochgeladen werden.'
+    file-too-large: "Die hochgeladene Datei ist zu gross."
```

Comment:
Would it make sense to give an estimate on how large a file can be?

Reply:
I thought about that but explicitly decided against it:

1. We can't read it from the response
2. Configuring it seems weird as we'd have some option called `maxUploadSize: "1MB"` or `maxUploadSize: 1000` and then compute it to a human readable format
3. The host app can simply override the translation :)